documentation/modules/configuring/proc-config-mirrormaker2-replication.adoc

```diff
@@ -234,10 +236,11 @@ To configure topic offset synchronization, this property must also be set for th
 <39> xref:con-common-configuration-healthchecks-reference[Healthchecks] to know when to restart a container (liveness) and when a container can accept traffic (readiness).
 <40> xref:con-common-configuration-jvm-reference[JVM configuration options] to optimize performance for the Virtual Machine (VM) running Kafka MirrorMaker.
 <41> ADVANCED OPTION: xref:con-common-configuration-images-reference[Container image configuration], which is recommended only in special situations.
-<42> xref:assembly-customizing-kubernetes-resources-str[Template customization]. Here a pod is scheduled with anti-affinity, so the pod is not scheduled on nodes with the same hostname.
-<43> Environment variables are also xref:ref-tracing-environment-variables-str[set for distributed tracing using Jaeger].
-<44> xref:assembly-distributed-tracing-str[Distributed tracing is enabled for Jaeger].
-<45> xref:type-ExternalConfiguration-reference[External configuration] for a Kubernetes Secret mounted to Kafka MirrorMaker as an environment variable.
+<42> xref:type-Rack-reference[Rack awareness] is configured to spread replicas across different racks. A `topologykey` must match the label of a cluster node.
```

Comment:
I would say: "across different racks or availability zones" and "must match a node label containing the rack id, like the well-known [topology.kubernetes.io/zone](https://kubernetes.io/docs/reference/labels-annotations-taints/#topologykubernetesiozone)."